sdk/keyvault/azure-security-keyvault-secrets/src/secret_client.cpp

```diff
@@ -25,7 +25,33 @@ using namespace Azure::Core::Http::Policies::_internal;
 
 namespace {
 constexpr static const char TelemetryName[] = "keyvault-secrets";
+
+struct RequestWithContinuationToken final
+{
+  std::vector<std::string> Path;
+  std::unique_ptr<std::map<std::string, std::string>> Query;
+};
+
+static inline RequestWithContinuationToken BuildRequestFromContinuationToken(
```

Comment:
I removed this method on Key Vault Keys, in case you want to take  a look to latest `main`.  Don't need to make the change now, however.